### ISSUE TITLE
User is removed from a meeting, when assigning them to the default group

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-backend#882]
I sent this payload:
```json
[
  {
    "action":"meeting.update",
    "data":[
      {
        "name":"Der Zehnte",
        "description":"",
        "location":"",
        "start_time":1627596000,
        "end_time":1627596000,
        "organization_tag_ids":[
          
        ],
        "jitsi_domain":"",
        "jitsi_room_name":"",
        "jitsi_room_password":"",
        "id":21
      }
    ]
  },
  {
    "action":"user.update",
    "data":[
      {
        "id":2,
        "group_$_ids":{
          "21":[
            101
          ]
        }
      }
    ]
  }
]
```

Why is then the user removed from the meeting `meeting/21`?
See logs: 
```
backend_1           | DEBUG:openslides_backend.services.datastore.adapter:Start WRITE request to datastore with the following data: Write request: [WriteRequest(events=[{'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('meeting/21'), 'fields': {'name': 'Der Zehnte', 'description': '', 'location': '', 'start_time': 1627596000, 'end_time': 1627596000, 'organization_tag_ids': [], 'jitsi_domain': '', 'jitsi_room_name': '', 'jitsi_room_password': ''}}], information={FullQualifiedId('meeting/21'): ['Object updated']}, user_id=1, locked_fields={'meeting/21/committee_id': 117, 'user/1/committee_$1_management_level': 95, 'user/1/organization_management_level': 95, 'meeting/21/organization_tag_ids': 117}), WriteRequest(events=[{'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('group/102'), 'fields': {'user_ids': [1, 6]}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('meeting/21'), 'list_fields': {'remove': {'user_ids': [2]}}}, {'type': <EventType.Update: 'update'>, 'fqid': FullQualifiedId('user/2'), 'fields': {'group_$_ids': ['1', '7', '20', '21'], 'group_$21_ids': [101]}}], information={FullQualifiedId('group/102'): ['Object attachment to group reset'], FullQualifiedId('meeting/21'): ['Object updated'], FullQualifiedId('user/2'): ['Object updated']}, user_id=1, locked_fields={'user/1/committee_$_management_level': 95, 'user/1/group_$_ids': 95, 'user/1/committee_ids': 95, 'user/1/organization_management_level': 95, 'user/2/group_$_ids': 117, 'group/101/meeting_id': 117, 'user/2/group_$21_ids': 117, 'group/102/user_ids': 115, 'group/102/meeting_id': 115})]
```

Database:
```json
{
  "id":2,
  "email":"",
  "title":"",
  "gender":"female",
  "number_$":[
    "1"
  ],
  "password":"316af7b2ddc20ead599c38541fbe87e9a9e4e960d4017d6e59de188b41b2758fDB3tv5HcCtPRREt7bPGqerTf1AbmoKXt/fVFkLY4znDRh2Yy0m3ZjXD0nHI8oa6KrGlHH/cvysfvf8i2fWIzmw==",
  "username":"a",
  "comment_$":[
    "1"
  ],
  "is_active":true,
  "last_name":"",
  "number_$1":"12345-67891",
  "about_me_$":[
    "1"
  ],
  "comment_$1":"Test comment a",
  "first_name":"a",
  "vote_$_ids":[
    
  ],
  "about_me_$1":"What I want to say about me with a",
  "group_$_ids":[
    "1",
    "7",
    "20",
    "21"
  ],
  "meeting_ids":[
    1,
    7,
    20,
    21
  ],
  "group_$1_ids":[
    5
  ],
  "group_$7_ids":[
    31
  ],
  "is_demo_user":false,
  "meta_deleted":false,
  "option_$_ids":[
    "1"
  ],
  "committee_ids":[
    1
  ],
  "group_$20_ids":[
    97
  ],
  "group_$21_ids":[
    101
  ],
  "meta_position":111,
  "option_$1_ids":[
    9,
    12
  ],
  "speaker_$_ids":[
    "1"
  ],
  "vote_weight_$":[
    "1"
  ],
  "default_number":"",
  "speaker_$1_ids":[
    2,
    3,
    7,
    10,
    11,
    13
  ],
  "vote_weight_$1":"1.000000",
  "default_password":"a",
  "poll_voted_$_ids":[
    
  ],
  "projection_$_ids":[
    
  ],
  "structure_level_$":[
    "1"
  ],
  "is_physical_person":true,
  "structure_level_$1":"Test structure level a",
  "default_vote_weight":"1.000000",
  "personal_note_$_ids":[
    
  ],
  "submitted_motion_$_ids":[
    "1"
  ],
  "supported_motion_$_ids":[
    
  ],
  "vote_delegated_$_to_id":[
    
  ],
  "can_change_own_password":true,
  "default_structure_level":"",
  "submitted_motion_$1_ids":[
    5
  ],
  "is_present_in_meeting_ids":[
    
  ],
  "vote_delegated_vote_$_ids":[
    
  ],
  "assignment_candidate_$_ids":[
    "1"
  ],
  "assignment_candidate_$1_ids":[
    3,
    5
  ],
  "vote_delegations_$_from_ids":[
    
  ],
  "committee_$_management_level":[
    
  ]
}
```

But there is no user `user/2` in the meeting `meeting/21`:
```json
{
  "id":21,
  "name":"Der Zehnte",
  "end_time":1627596000,
  "location":"",
  "user_ids":[
    1,
    6
  ],
  "group_ids":[
    101,
    102,
    103,
    104,
    105
  ],
  "start_time":1627596000,
  "description":"",
  "committee_id":1,
  "jitsi_domain":"",
  "meta_deleted":false,
  "welcome_text":"Space for your welcome text.",
  "welcome_title":"Welcome to OpenSlides",
  "admin_group_id":102,
  "default_group_id":101,
 [...]
```

**Comment by jsangmeister** [comment]
I think this is the same error as in #869 and should be fixed with #875, which is merged now. Can you try again with the new code?

**Comment by GabrielInTheWorld** [comment]
It works as expected, now
